### DISCUSSION TITLE
reporting a site bug

### DISCUSSION THREAD

**Discussion by luis-orantes** [codewars/codewars.com#3015] [General]
How I do I report a bug in the site. I think is not in a particular case but about the site.

**Comment by hobovsky** [comment]
- You can come to Codewars Discord and let us know what you found in the `#codewars` channel, or in `#contact-mods` channel, or
- You can create a ticket on the [Codewars GitHub issues board](https://github.com/codewars/codewars.com/issues), or
- You can just write here and the report will be dispatched and handled accordingly.
